datalake/src/main/java/com/sequenceiq/datalake/flow/upgrade/database/handler/SdxUpgradeDatabaseServerHandler.java

```diff
@@ -0,0 +1,61 @@
+package com.sequenceiq.datalake.flow.upgrade.database.handler;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.common.event.Selectable;
+import com.sequenceiq.datalake.entity.SdxCluster;
+import com.sequenceiq.datalake.flow.upgrade.database.event.SdxUpgradeDatabaseServerFailedEvent;
+import com.sequenceiq.datalake.flow.upgrade.database.event.SdxUpgradeDatabaseServerSuccessEvent;
+import com.sequenceiq.datalake.flow.upgrade.database.event.UpgradeDatabaseServerRequest;
+import com.sequenceiq.datalake.service.sdx.SdxService;
+import com.sequenceiq.datalake.service.upgrade.database.SdxDatabaseServerUpgradeService;
+import com.sequenceiq.flow.event.EventSelectorUtil;
+import com.sequenceiq.flow.reactor.api.handler.ExceptionCatcherEventHandler;
+import com.sequenceiq.flow.reactor.api.handler.HandlerEvent;
+
+import reactor.bus.Event;
+
+@Component
+public class SdxUpgradeDatabaseServerHandler extends ExceptionCatcherEventHandler<UpgradeDatabaseServerRequest> {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(SdxUpgradeDatabaseServerHandler.class);
+
+    @Inject
+    private SdxDatabaseServerUpgradeService sdxDatabaseServerUpgradeService;
+
+    @Inject
+    private SdxService sdxService;
+
+    @Override
+    public String selector() {
+        return EventSelectorUtil.selector(UpgradeDatabaseServerRequest.class);
+    }
+
+    @Override
+    protected Selectable defaultFailureEvent(Long resourceId, Exception e, Event<UpgradeDatabaseServerRequest> event) {
+        LOGGER.warn("Exception during upgrade of database server in SDX: ", e);
+        return new SdxUpgradeDatabaseServerFailedEvent(resourceId, event.getData().getUserId(), e, "");
```

Comment:
why is the last param empty instead of passing the exception message? 
Furthermore, what is `com.sequenceiq.datalake.flow.upgrade.database.event.SdxUpgradeDatabaseServerFailedEvent#message` used for?

Reply:
The message is used in the WaitHandler: the poller has some custom message and this would have been the way to transmit it. On the other hand, it was not used. Also, most exceptions will come with a message, so not clear if it would be useful.